### PR TITLE
Add refiners as a library

### PR DESCRIPTION
**Pull request by Laurent2916**
This PR adds https://github.com/finegrain-ai/refiners as a library on the Hub.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -408,6 +408,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"refiners": {
+		prettyLabel: "Refiners",
+		repoName: "refiners",
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:413]:
```suggestion
		repoName: "Refiners",
```

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -408,6 +408,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
+		repoUrl: "https://github.com/finegrain-ai/refiners",
+		docsUrl: "https://refine.rs/",
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:417]: For what I can see in [juggernaut.reborn](https://huggingface.co/refiners/juggernaut.reborn/tree/main), [refiners/juggernaut.aftermath](https://huggingface.co/refiners/juggernaut.aftermath/tree/main) and [refiners/juggernaut.inpainting](https://huggingface.co/refiners/juggernaut.inpainting/tree/main), the model repos always contain the same three safetensors files. When using the `refiners` library, which file is necessarily downloaded? Is it the three of them? If yes, the risk is to count 3x the number of downloads hence artificially boosting the counter without relevant usage. What we suggest to do in this case is to count downloads only on 1 file, for example `countDownloads: `path:"text_encoder.safetensors"`,`. WDYT?

**Review by Wauplin — COMMENTED** [review]
Thanks for the PR @Laurent2916! It looks good to me. I simply have a question related to how to count downloads.

Also, would you be interested in generating a code snippet to display to users how to use your models? It's not an obligation but it usually helps getting usage on a model.

File: packages/tasks/src/model-libraries.ts (hunk 2 above)
Comment (Laurent2916) [reply packages/tasks/src/model-libraries.ts:417]: Hi @Wauplin!
:thinking: Thanks I foresee that. I don't think we plan on supporting an "automagic" method like diffusers' ``

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:417]: Sounds good! Splitting repos is usually better to document things independently + monitor usage. Just let me know then :)

Comment (Laurent2916) [reply packages/tasks/src/model-libraries.ts:417]: Hi @Wauplin, I finally found the time to split the models into their own repo!
Should I squash the commits of this PR?

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:417]: Thanks @Laurent2916! This is much better like this :) Now [all refiners models](https://huggingface.co/models?other=refiners) have a `model.safetensors` file so let's count downloads on this file only.

```suggestion
		countDownloads: `path:"model.safetensors"`,
```

**Review by Wauplin — APPROVED** [review]
Thanks for the changes and cleaning the repos @Laurent2916! I just reviewed everything and it looks good to me :rocket: I've left a minor comment about how to count downloads (better to use path than path_extension) but apart from that, everything seems ready! I'll merge the PR now. It should be deployed in the coming days.

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -435,6 +435,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"refiners": {
+		prettyLabel: "Refiners",
+		repoName: "Refiners",
+		repoUrl: "https://github.com/finegrain-ai/refiners",
+		docsUrl: "https://refine.rs/",
+		filter: false,
+		countDownloads: `path:"model.safetensors"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:444]:
```suggestion
		countDownloads: `path:"model.safetensors"`,
	},	
```

File: packages/tasks/src/model-libraries.ts [hunk 4]
```diff
@@ -435,6 +435,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"refiners": {
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:438]:
```suggestion
	refiners: {
```

File: packages/tasks/src/model-libraries.ts [hunk 5]
```diff
@@ -435,6 +435,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	refiners: {
+		prettyLabel: "Refiners",
+		repoName: "Refiners",
+		repoUrl: "https://github.com/finegrain-ai/refiners",
+		docsUrl: "https://refine.rs/",
+		filter: false,
+		countDownloads: `path:"model.safetensors"`,
+	},
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:445]:
```suggestion
	},
```

**Pull request merged**